public/scripts/extensions/quick-reply/html/qrEditor.html

```diff
@@ -59,48 +59,48 @@ <h3>Context Menu</h3>
 		</div>
 
 
-		<h3>Auto-Execute</h3>
+		<h3 data-i18n="Auto-Execute">Auto-Execute</h3>
 		<div class="flex-container flexFlowColumn">
 			<label class="checkbox_label" title="Prevent this quick reply from triggering other auto-executed quick replies while auto-executing (i.e., prevent recursive auto-execution)">
 				<input type="checkbox" id="qr--preventAutoExecute" >
-				<span><i class="fa-solid fa-fw fa-plane-slash"></i> Don't trigger auto-execute</span>
+				<span><i class="fa-solid fa-fw fa-plane-slash"></i><div data-i18n="Don't trigger auto-execute">Don't trigger auto-execute</div></span>
```

Comment:
Should these be div and not span?

Reply:
fixed by https://github.com/SillyTavern/SillyTavern/pull/2376/commits/3c696c64b341436768746863591250da6c336932